config/initializers/doorkeeper.rb

```diff
@@ -45,7 +45,7 @@
   # Optional parameter :confirmation => true (default false) if you want to enforce ownership of
   # a registered application
   # Note: you must also run the rails g doorkeeper:application_owner generator to provide the necessary support
-  # enable_application_owner :confirmation => true
+  enable_application_owner
```

Comment:
Yes, it sets the application owner to the current user if that data is available. Without that, we can't associate apps with owners.